panel/io/resources.py

```diff
@@ -258,7 +258,8 @@ def component_resource_path(component, attr, path):
     component_path = COMPONENT_PATH
     if state.rel_path:
         component_path = f"{state.rel_path}/{component_path}"
-    rel_path = os.fspath(resolve_custom_path(component, path, relative=True)).replace(os.path.sep, '/')
+    custom_path = resolve_custom_path(component, path, relative=True)
+    rel_path = os.fspath(custom_path).replace(os.path.sep, '/') if custom_path else ""
```

Comment:
I think this should never happen, so an error SHOULD be raised here.

Reply:
`resolve_custom_path` returning `None`? It's exactly what happens if you use "Preview with Panel in New Browser tab" with notifications enabled. See the issue linked in the description: https://github.com/holoviz/panel/issues/7380

Reply:
That's just a consequence of a different bug, that I fixed in this PR.

Reply:
Thanks, my fix was not quite correct.